src/main/java/com/cag/cagbackendapi/entities/UserEntity.java

```diff
@@ -31,8 +30,10 @@
     public String email;
     public Boolean active_status;
     public String session_id;
+    public String img_url;
+    public Boolean agreed_18;
 
-    public UserResponseDto toDto() {
-        return new UserResponseDto(this.userId, this.first_name, this.last_name, this.email, this.active_status, this.session_id);
+    public UserDto toDto() {
+        return new UserDto(this.userId, this.first_name, this.last_name, this.email, this.active_status, this.session_id, this.img_url, this.agreed_18);
```

Comment:
agreed_18 must be true to register the user and we don't store birthdate or age on the User table. We do store age on Profile but only in age increments that they are comfortable casting for - different from their real age. At the time in the application where we do the "Are you 18" validation - they won't have a profile yet, which is why "agreed_18" belongs on the Users table since the Profile doesn't exist. There are a lot of tiny business rules but if you look at the flow diagram it makes a little more sense.

Reply:
It is a useless field we need to carry around in the application because after they sign up for the app - there is literally no purpose for the field. We store it and carry it around in our app so we can show a judge if anyone testifies against us.